From 559e21bd7ba098cd007af7add55ad2e537d2f936 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Wed, 15 Aug 2018 17:30:23 +0100
Subject: [PATCH 223/249] lineage: qcom: Fork newest custom audio policy HAL

 * This HAL should be pretty much SoC-agnostic,
   so let's just switch to a unique project.

Change-Id: I509ab67346eabd0679d32bab3c9def5e2840670f
---
 snippets/lineage.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 4c14048..88bbfea 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -103,6 +103,7 @@
   <!--<project path="vendor/nxp/opensource/external/libnfc-nci" name="LineageOS/android_vendor_nxp_opensource_external_libnfc-nci" />-->
   <!--<project path="vendor/nxp/opensource/frameworks" name="LineageOS/android_vendor_nxp_opensource_frameworks" />-->
   <!--<project path="vendor/nxp/opensource/packages/apps/Nfc" name="LineageOS/android_vendor_nxp_opensource_packages_apps_Nfc" />-->
+  <project path="vendor/qcom/opensource/audio" name="LineageOS/android_vendor_qcom_opensource_audio" />
   <!--<project path="vendor/qcom/opensource/cryptfs_hw" name="LineageOS/android_vendor_qcom_opensource_cryptfs_hw" />-->
   <!--<project path="vendor/qcom/opensource/dataservices" name="LineageOS/android_vendor_qcom_opensource_dataservices" />-->
   <project path="vendor/qcom/opensource/interfaces" name="LineageOS/android_vendor_qcom_opensource_interfaces" />
-- 
2.17.1

